### PR TITLE
[PL-47646] PL 1.26.11 Release Notes

### PR DESCRIPTION
**Pull request by saifeemustafaqharness**
[PL-47646](https://harness.atlassian.net/browse/PL-47646)

https://harness.atlassian.net/wiki/spaces/PD/pages/21663350825/NG+Release+SAAS+February+19th+2024+for+1.26.x-+Build

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screenshot. 

[PL-47646]: https://harness.atlassian.net/browse/PL-47646?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by saifeemustafaqharness** [comment]
@brian-f-harness Information about other tickets that you were asked above can be found in the conversation in the following ticket https://harness.atlassian.net/browse/PL-47646

**Comment by brian-f-harness** [comment]
> Suggest removing the three items published with 1.25.5 (PL-46595, PL-46492, and PL-42600) that are on the 1.26.x list for the delegate release from the existing PL 1.25.5 RN. 1.25.5 wasn't a delegate release week, and they have the Affects-delegate label. These items didn't include the standard delegate version dependency verbiage in the published 1.25.5 RN because it wasn't a delegate release week.
> 
> We should include these items with this PR for 1.26.x after moving them from 1.25.x. We'd want to add the standard delegate dependency note (This item requires Harness Delegate version...).

@saifeemustafaqharness These tickets are not mentioned in the conversation on the ticket. Please work with the delegate team to determine how to resolve this issue. We documented three items in the last set of PL release notes that shouldn't have been on the list. They're on the list for 1.26.11 because it's a delegate release week. We should remove these items from the published 1.25.5 release and add them to the 1.26.11 RN with the standard delegate dependency verbiage according the the process for delegate RN established with @raghuAtWings early 2023.

**Comment by saifeemustafaqharness** [comment]
@brian-f-harness I have moved (PL-46595, PL-46492, and PL-42600) from 1.25.5 to 1.26.11

Also please note the following:
PL-46492 is also present in 1.24.7
PL-42600 is also present in 1.19.6

cc: @nataraja2018 Can you please review the changes as well. Thanks

**Comment by brian-f-harness** [comment]
> PL-46595

Thank you, @saifeemustafaqharness. I didn't know the following were published:
PL-46492 in 1.24.7
PL-42600 in 1.19.6

Maybe we only need PL-46595 in 1.26.11 if the other two items were reopened and then later closed or closed after the release? This is the jira monitoring issue called out in the open RN item issues bug.

File: release-notes/platform.md [hunk 1]
```diff
@@ -82,6 +82,32 @@ The following deprecated API endpoints are longer supported:
...
+
+   Implemented LDAP to perform paginated queries by default for large groups, with a fallback to non-paginated calls, ensuring complete user synchronisation.
+
+   This item is available with Harness Platform version 1.26.11 and does not require a new delegate version. For information about Harness Delegate features that require a specific delegate version, go to the Delegate release notes.
+
+- Pipelines were failing due to errors related to the inability to acquire delegate tasks. (PL-42600, ZD-54025, ZD-54324)
+
+   The logic for calculating CPU and Memory usage has been improved, specifically for scenarios utilizing the dynamic task request handling feature in delegates, enhancing the reliability of task allocation and pipeline execution.
+
+   This item is available with Harness Platform version 1.26.11 and does not require a new delegate version. For information about Harness Delegate features that require a specific delegate version, go to the Delegate release notes.
```
Comment (brian-f-harness) [inline release-notes/platform.md:109]:
```suggestion
   This item requires Harness Delegate version 24.02.82402. For information about Harness Delegate features that require a specific delegate version, go to the Delegate release notes.
```

File: release-notes/platform.md [hunk 2]
```diff
@@ -82,6 +82,32 @@ The following deprecated API endpoints are longer supported:
...
+   
+   The retry interval has now been modified to increase by a factor of 2 times the number of failures. Consequently, after the first failure, the second attempt will occur after a 2-second delay, and the third attempt will be made after a 4-second delay, enhancing the robustness of secret management operations.
+
+   This item is available with Harness Platform version 1.26.11 and does not require a new delegate version. For information about Harness Delegate features that require a specific delegate version, go to the Delegate release notes.
+
+- When linking an SSO group with over 1,000 users, only 1,000 users were syncing in Harness due to a limitation with LDAP groups syncing. (PL-46492, ZD-56741)
+
+   Implemented LDAP to perform paginated queries by default for large groups, with a fallback to non-paginated calls, ensuring complete user synchronisation.
+
+   This item is available with Harness Platform version 1.26.11 and does not require a new delegate version. For information about Harness Delegate features that require a specific delegate version, go to the Delegate release notes.
```
Comment (brian-f-harness) [inline release-notes/platform.md:103]:
```suggestion
   This item requires Harness Delegate version 24.02.82402. For information about Harness Delegate features that require a specific delegate version, go to the Delegate release notes.
```

File: release-notes/platform.md [hunk 3]
```diff
@@ -82,6 +82,32 @@ The following deprecated API endpoints are longer supported:
...
+
+- Creating or updating a project with an org identifier absent in the account used to throw error code: 500. (PL-47059, ZD-58093)
+
+   Now, the UI displays the following improved error message if the provided org identifier is absent: `Organisation with identifier "OrgId" does not exist in accountIdentifier: "accountId"`.
+
+- The retry interval for attempting to create or read secrets from HashiCorp Vault was fixed at 1 second after each failure. (PL-46595, ZD-57053)
+   
+   The retry interval has now been modified to increase by a factor of 2 times the number of failures. Consequently, after the first failure, the second attempt will occur after a 2-second delay, and the third attempt will be made after a 4-second delay, enhancing the robustness of secret management operations.
+
+   This item is available with Harness Platform version 1.26.11 and does not require a new delegate version. For information about Harness Delegate features that require a specific delegate version, go to the Delegate release notes.
```
Comment (brian-f-harness) [inline release-notes/platform.md:97]:
```suggestion
   This item requires Harness Delegate version 24.02.82402. For information about Harness Delegate features that require a specific delegate version, go to the Delegate release notes.
```

**Review by brian-f-harness — CHANGES_REQUESTED** [review]
Items in the delegate dependent section require:
This item requires Harness Delegate version 24.02.82402. 

Items with delegate in the component, but without the Affects-delegate or Affects_delegate labels require:
This item is available with Harness Platform version 1.26.11 and does not require a new delegate version.

**Comment by saifeemustafaqharness** [comment]
> > PL-46595
> 
> Thank you, @saifeemustafaqharness. I didn't know the following were published: PL-46492 in 1.24.7 PL-42600 in 1.19.6
> 
> Maybe we only need PL-46595 in 1.26.11 if the other two items were reopened and then later closed or closed after the release? This is the jira monitoring issue called out in the open RN item issues bug.
> 
> 1.19.6 wasn't a delegate release. Do we need to move PL-42600 to 1.20.9 and update the 24.01.82108 with the item? It isn't in delegate RN but has the Affects-delegate label and is on this list for delegate-dependent items.

Maybe @nataraja2018 can provide a solid answer to this, what should we do @nataraja2018 ?

**Review by brian-f-harness — APPROVED** [review]

**Comment by brian-f-harness** [comment]
> > Items in the delegate dependent section require: This item requires Harness Delegate version 24.02.82402.
> > Items with delegate in the component, but without the Affects-delegate or Affects_delegate labels require: This item is available with Harness Platform version 1.26.11 and does not require a new delegate version.
> 
> Okay, here's my understanding: I have to check if delegate component is involved then add "This item is available with Harness Platform version 1.26.11 and does not require a new delegate version."
> 
> And if any ticket is there in the lower column of "NG Delegate dependent changes" then add "This item requires Harness Delegate version 24.02.82402"
> 
> cc @nataraja2018 for visibility.

@saifeemustafaqharness Correct. This is the process for delegate items in PL RN. 

For items under the NG Delegate dependent changes, always use the "This item requires Harness Delegate version 24.02.82402." verbiage. (Some of these items may also be delegate component, but the Affects-delegate and Affects_delegate labels determine the items shown under NG Delegate dependent changes. Even if these items are also delegate component, we want to use the requires delegate version sentence.)

For items in the main list with delegate component, use: "This item is available with Harness Platform version 1.26.11 and does not require a new delegate version."

File: release-notes/platform.md [hunk 4]
```diff
@@ -82,6 +82,32 @@ The following deprecated API endpoints are longer supported:
...
+
+- Creating or updating a project with an org identifier absent in the account used to throw error code: 500. (PL-47059, ZD-58093)
+
+   Now, the UI displays the following improved error message if the provided org identifier is absent: `Organisation with identifier "OrgId" does not exist in accountIdentifier: "accountId"`.
+
+- The retry interval for attempting to create or read secrets from HashiCorp Vault was fixed at 1 second after each failure. (PL-46595, ZD-57053)
+   
+   The retry interval has now been modified to increase by a factor of 2 times the number of failures. Consequently, after the first failure, the second attempt will occur after a 2-second delay, and the third attempt will be made after a 4-second delay, enhancing the robustness of secret management operations.
+
+   This item requires Harness Delegate version 24.02.82402. For information about Harness Delegate features that require a specific delegate version, go to the Delegate release notes.
```
Comment (brian-f-harness) [inline release-notes/platform.md:97]:
```suggestion
   This item requires Harness Delegate version 24.02.82402. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```

File: release-notes/platform.md [hunk 5]
```diff
@@ -82,6 +82,32 @@ The following deprecated API endpoints are longer supported:
...
+
+- When linking an SSO group with over 1,000 users, only 1,000 users were syncing in Harness due to a limitation with LDAP groups syncing. (PL-46492, ZD-56741)
+
+   Implemented LDAP to perform paginated queries by default for large groups, with a fallback to non-paginated calls, ensuring complete user synchronisation.
+
+   This item requires Harness Delegate version 24.02.82402. For information about Harness Delegate features that require a specific delegate version, go to the Delegate release notes.
```
Comment (brian-f-harness) [inline release-notes/platform.md:103]:
```suggestion
   This item requires Harness Delegate version 24.02.82402. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```

File: release-notes/platform.md [hunk 6]
```diff
@@ -82,6 +82,32 @@ The following deprecated API endpoints are longer supported:
...
+
+- Pipelines were failing due to errors related to the inability to acquire delegate tasks. (PL-42600, ZD-54025, ZD-54324)
+
+   The logic for calculating CPU and Memory usage has been improved, specifically for scenarios utilizing the dynamic task request handling feature in delegates, enhancing the reliability of task allocation and pipeline execution.
+
+   This item requires Harness Delegate version 24.02.82402. For information about Harness Delegate features that require a specific delegate version, go to the Delegate release notes.
```
Comment (brian-f-harness) [inline release-notes/platform.md:109]:
```suggestion
   This item requires Harness Delegate version 24.02.82402. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```

**Pull request merged**